code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -415,3 +415,6 @@ h1.alert, h2.alert		{color: #000000;}
 /* HELPER CLASSES */
 .text-normal {font-weight: normal; font-style: normal;}
 .hidden {display: none; visibility: hidden;}
+
+/* EMOJI STUFF */
+.emoji {max-height: 16px; max-width: 16px}
```

Comment:
you might want to use ems rather than pixels so that when users change their font size the emoji size changes along with it.

see the css for icon for an example.